playbooks/roles/kibana/templates/nginx.conf.j2

```diff
@@ -0,0 +1,47 @@
+upstream elasticsearch_server {
```

Comment:
All of the other nginx templates are in `playbooks/roles/nginx/templates/edx/app/nginx/sites-available/<SITE_NAME_HERE>.j2`. Barring a compelling reason to the contrary, we should follow convention.